src/provisioning/api.md

```diff
@@ -0,0 +1,167 @@
+Provisioning API for matrix-hookshot
```

Comment:
endpoint to get the bot's user ID (for invites/power level) please 😇 

I believe this will be per-service, but happy to be wrong :D

Reply:
At the moment we use one bot user for all, but I suppose we could start spouting special users for each room.

Reply:
The API now exposes a botUserId for all connections. There is also a connectiontypes api to help you search for what's enabled on the bridge.

Reply:
not sure where the resolve button is on this thread, but the thread is resolved - thank you!